health/utils/constants.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+// List of constant keys and values for health framework
+const (
+	SourceTypeKey     = "source-type"
+	DefaultSourceType = "zengo.health" // ZEN+NG+GO ZENoss NewGeneration GO :)
```

Comment:
Nice try ;)
But per our convention, something like  zenoss.collection.health may be more apropos
Strictly speaking, it will be whatever source-type the collector and/or plugin has set + a ".health" suffix. We will need to figure out how to do that at.  the zdm level for example.

Reply:
ok. FYI: even now we have an ability to override it